requirements-colab.txt

```diff
@@ -1,21 +1,21 @@
 # Requirements for use inside Google Colab, where Jupyter environment is already present
 
-appnope==0.1.2
-ase==3.21.1
-async-generator==1.10
-backcall==0.2.0
-cycler==0.10.0
-exabyte-api-client==2021.1.18
-jedi==0.18.0
-kiwisolver==1.3.1
-monty==2021.3.3
-mpmath==1.2.1
-nest-asyncio==1.5.1
-networkx==2.5.1
-palettable==3.3.0
-pymatgen==2021.2.16
-retrying==1.3.3
-ruamel.yaml==0.17.4
-ruamel.yaml.clib==0.2.2
-spglib==1.16.1
-uncertainties==3.1.5
+appnope
+ase
+async-generator
+backcall
+cycler
+exabyte-api-client
+jedi
+kiwisolver
+monty
+mpmath
+nest-asyncio
+networkx
+palettable
+pymatgen
+retrying
+ruamel.yaml
+ruamel.yaml.clib
+spglib
+uncertainties
```

Comment:
What i meant earlier is that we don't really need all these packages - instead of relaxing the versions (which I think we should have fixed), we should only have the packages that we actually require. For instance, `ruamel.yaml` and `spglib` and `monty` are all dependencies of pymatgen.

I think, the list should only contain:

- exabyte-api-client
- ase
- pymatgen
- maybe one or two other entries

Reply:
Maybe pandas

Reply:
+rdkit